### ISSUE TITLE
Не відображаються аватари користувачив

### ISSUE THREAD

**Issue by p-butovets** [mblog-dev/wish#3]
Скрін: https://prnt.sc/5MQrTSDU3Ynw

**Comment by KostyaTretyak** [comment]
Коли таке було, і чи зникало після перезавантаження?

**Comment by KostyaTretyak** [comment]
Можливо у вас працює блокер реклами, наприклад, AdBlock чи якийсь аналог. Можливо це його робота. До речі, його можна вимкнути бо на mblog.dev немає і не буде нав'язливої реклами.

**Comment by p-butovets** [comment]
так, эмоджі я не бачив:) налаштування браузера стандартні, без блокувань і тд

**Comment by KostyaTretyak** [comment]
А зараз емоджі показуються?

**Comment by p-butovets** [comment]
Наразі все лягло https://prnt.sc/M4FGQZjJhDt5

**Comment by p-butovets** [comment]
> А зараз емоджі показуються тут https://mblog.dev/post/1?

Майже всі https://prnt.sc/aagnxaUkekrW :))

**Comment by KostyaTretyak** [comment]
Ви регулярно оновлюєте Windows? Чи вона у вас... така що не можна оновлювати? Ось тут пишуть про проблеми із емоджі на Windows https://dou.ua/forums/topic/35325/

**Comment by KostyaTretyak** [comment]
Цікаво чи бачите ви емоджі на ДОУ, наприклад тут https://dou.ua/forums/topic/34491/#2208426
